app/hotels/src/singleHotel/paymentScreen/PaymentWebView.js

```diff
@@ -0,0 +1,65 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import {
+  WebView,
+  GeneralError,
+  type WebViewStateChangeEvent,
+  Logger,
+} from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+import {
+  withHotelsContext,
+  type HotelsContextState,
+  type ApiProvider,
+} from '../../HotelsContext';
+
+type Props = {|
+  +url: ?string,
+  +apiProvider: ApiProvider,
+|};
+
+export class PaymentWebView extends React.Component<Props> {
+  onNavigationStateChange = (event: WebViewStateChangeEvent) => {
+    if (!event.loading && event.url.includes('booking.com/confirmation')) {
```

Comment:
So it's still the same URL. This would probably need a comment to explain why it's the case? Otherwise it looks like it's a mistake.